parachain/Cargo.toml

```diff
@@ -101,7 +101,10 @@ snowblink-runtime-benchmarks = [ "snowblink-runtime/runtime-benchmarks" ]
 snowbase-runtime-benchmarks = [ "snowbase-runtime/runtime-benchmarks" ]
 polkadot-runtime-benchmarks = [ "polkadot-service/runtime-benchmarks" ]
 
-runtime-benchmarks = []
+runtime-benchmarks = [
+	"frame-benchmarking/runtime-benchmarks",
```

Comment:
Why do we need the `/runtime-benchmarks` suffix here? We're using `frame-benchmarking` in the runtime-benchmarks feature for the runtimes.